imagetest/test_suites/image_boot/image_boot_test.go

```diff
@@ -55,6 +55,10 @@ func TestGuestSecureBoot(t *testing.T) {
 		t.Skip("secure boot is not supported on Debian 9")
 	}
 
+	if strings.Contains(image, "rocky-linux-8") {
```

Comment:
i was gonna make a suggested change but github won't let me make suggestions on unchanged lines. i think this is now the time to make a case statement

Reply:
Its somewhat temporary- within a month (I suspect) we'll be back to one case. Still want to do that?

Reply:
fine either way, then